libsolidity/interface/OptimiserSettings.h

```diff
@@ -45,7 +45,7 @@ struct OptimiserSettings
 			"xarulrul"                 // Prune a bit more in SSA
 			"xarrcL"                   // Turn into SSA again and simplify
 			"gvif"                     // Run full inliner
-			"CTUcarrLsTOtfDncarrIulc"  // SSA plus simplify
+			"CTUcarrLsTFOtfDncarrIulc" // SSA plus simplify
```

Comment:
Can you explain why you put the F in exactly that place?

Reply:
Mainly I wanted to have it after the full inliner.